repository/src/main/java/org/alfresco/repo/event2/NodeResourceHelper.java

```diff
@@ -158,6 +160,15 @@ private boolean isSubClass(QName className, QName ofClassQName)
         return dictionaryService.isSubClass(className, ofClassQName);
     }
 
+    private String getAssocQName(NodeRef nodeRef) {
```

Comment:
Does it make sense to change the naming a bit as the assoc type here is only the primary parent-child assoc? There are also secondary parent-child and peer assocs that we are planning to implement later on.

Reply:
Should I call it primaryAssocQName? Do you mean also in acs-event-model, not only the private method, right?

Reply:
yes, I think it should be ok to use "getPrimaryAssocQName"

Reply:
> Do you mean also in acs-event-model, not only the private method

I think it makes sense to change that only in NodeResource. I'll add the comment on that PR. I think we should replicate our V1 REST API.